Month1/Week2/Section2/Month1-Week2-Section2/Spotify/Song.cpp

```diff
@@ -0,0 +1,70 @@
+#include "Song.h"
+FSong::FSong()
+{
+    MinutesDuration = 0;
+    SecondsDuration = 0;
+    ArtistName = "";
+    SongName = "";
+}
+FSong::FSong(int EntryMinutesDuration, int EntrySecondsDuration, std::string EntryArtistName, std::string EntrySongName)
+{
+    MinutesDuration = EntryMinutesDuration;
+    SecondsDuration = EntrySecondsDuration;
+    ArtistName = EntryArtistName;
+    SongName = EntrySongName;
+
+}
+
+
```

Comment:
Remember that consistency is important. Here you have 3 spaces between functions, then you have none, then you have one, and then you have none for the rest of the file.

I consider one space to be the most legible and clear.

Reply:
Changed